app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsReactivationWorker.kt

```diff
@@ -0,0 +1,48 @@
+package org.wikipedia.suggestededits
+
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import org.wikipedia.Constants.INTENT_EXTRA_GO_TO_SE_TAB
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.auth.AccountUtil
+import org.wikipedia.main.MainActivity
+import org.wikipedia.notifications.NotificationPresenter
+import org.wikipedia.settings.Prefs
+import java.util.concurrent.TimeUnit
+
+
+class SuggestedEditsReactivationWorker(val context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
+
+    override fun doWork(): Result {
+        if (AccountUtil.isLoggedIn()) {
+            val days = System.currentTimeMillis() - Prefs.getLastDescriptionEditTime()
```

Comment:
I used `FIRST_NOTIFICATION_SHOW_ON_DAY = TimeUnit.DAYS.toMillis(3)` to convert the day to milliseconds, and it might cause confusion. I have changed it to a simpler solution to the latest commit.